scripts/check-siblings.js

```diff
@@ -1,12 +1,18 @@
 const fs = require('fs');
 const path = require('path');
-const { env } = require('process');
+const { env, exit } = require('process');
 
 const updateArgument = '--update-sentry-capacitor';
 
 // Filters all Sentry packages but Capacitor, CLI and Wizard.
 const jsonFilter = /\s*\"\@sentry\/(?!capacitor|wizard|cli|typescript|electron)(?<packageName>[a-zA-Z]+)\"\:\s*\"(?<version>.+)\"/;
 
+const IsE2E = env.sentry_e2e == 'true';
+
+function LogE2E(message) {
+  IsE2E && console.log("E2E_TEST: " + message);
+}
+
```

Comment:
Let's isolate the fix and remove the E2E test changes from this PR.

Reply:
Sounds good to me, I have removed the e2e logs on https://github.com/getsentry/sentry-capacitor/pull/829/commits/e5640b0f8e5e4caaa9634b35274d6432ab526c9e